prometheus-ksonnet/grafana/config.libsonnet

```diff
@@ -1,49 +1,35 @@
 {
   _config+:: {
-    admin_services+:: [
-      {
-        title: 'Grafana (Light)',
-        path: 'grafana',
-        params: '/?search=open&theme=light',
-        url: 'http://grafana.%(namespace)s.svc.%(cluster_dns_suffix)s/' % $._config,
-        allowWebsockets: true,
-      },
-      {
-        title: 'Grafana (Dark)',
-        path: 'grafana',
-        params: '/?search=open&theme=dark',
-        url: 'http://grafana.%(namespace)s.svc.%(cluster_dns_suffix)s/' % $._config,
-        allowWebsockets: true,
-      },
-    ],
-  },
+    // configure Grafana Lib based upon legacy configs below
+    rootUrl: $._config.grafana_root_url,
+    provisioningDir: $._config.grafana_provisioning_dir,
+    dashboardConfigMapCount: $._config.dashboard_config_maps,
```

Comment:
Not used?

Reply:
they are set further down in the file, supporting the legacy extension points.